src/components/Router/index.js

```diff
@@ -41,11 +41,23 @@ class CrossDeviceMobileRouter extends Component {
   }
 
   setConfig = (actions) => (data) => {
-    const {token, steps, documentType, step} = data
+    const {token, steps, documentType, step, woopraCookie} = data
+    this.setWoopraCookie(woopraCookie)
     this.setState({token, steps, step})
     actions.setDocumentType(documentType)
   }
 
+  setWoopraCookie = (cookie) => {
+    const cookie_name = window.onfidojssdkwoopra.config('cookie_name')
+    const cookie_expire = window.onfidojssdkwoopra.config('cookie_expire')
+    const cookie_path = window.onfidojssdkwoopra.config('cookie_path')
+    const cookie_domain = window.onfidojssdkwoopra.config('cookie_domain')
+    window.onfidojssdkwoopra.docCookies.setItem(
+      cookie_name, cookie, cookie_expire, cookie_path, cookie_domain
+    )
+    window.onfidojssdkwoopra.cookie = cookie
+  }
```

Comment:
cf. how the woopra JS SDK sets the cookie: https://github.com/onfido/js-client-tracker/blob/master/wpt.js#L885